src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -552,6 +643,90 @@ void EIRPlantLoopHeatPump::calcSourceSideHeatTransferASHP(EnergyPlusData &state)
     Real64 const CpSrc = Psychrometrics::PsyCpAirFnW(state.dataEnvrn->OutHumRat);
     Real64 const sourceMCp = this->sourceSideMassFlowRate * CpSrc;
     this->sourceSideOutletTemp = this->calcSourceOutletTemp(this->sourceSideInletTemp, this->sourceSideHeatTransfer / sourceMCp);
+    if (this->heatRecoveryAvailable && !this->heatRecoveryIsActive) {
+        // reset the HR report variables
+        this->heatRecoveryRate = 0.0;
+        this->heatRecoveryOutletTemp = this->heatRecoveryInletTemp;
+    }
```

Comment:
If heat recovery is not active, then set the heat recovery rates to `0` and reset the fluid outlet temperature to the inlet.